src/frontend/screens/Settings/components/Gamescope.tsx

```diff
@@ -0,0 +1,334 @@
+                  )}
+                />
+              }
+              onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                setGamescope({
+                  ...gamescope,
+                  gameHeight:
+                    setResolution(event.currentTarget.value) ??
+                    gamescope.gameHeight
+                })
+              }}
+            />
+          </div>
+          {/* Upscale Res */}
+          <div className="toggleRow">
+            <TextInputField
+              label={t('options.gamescope.upscaleWidth', 'Upscale Width')}
+              htmlId="upscaleWidth"
+              placeholder=""
+              maxLength={4}
+              value={gamescope.upscaleWidth}
+              afterInput={
+                <FontAwesomeIcon
+                  className="helpIcon"
+                  icon={faCircleInfo}
+                  title={t(
+                    'help.gamescope.upscaleWidth',
+                    'The width resolution used by gamescope. A 16:9 aspect ratio is assumed.'
+                  )}
+                />
+              }
+              onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                setGamescope({
+                  ...gamescope,
+                  upscaleWidth:
+                    setResolution(event.currentTarget.value) ??
+                    gamescope.upscaleWidth
+                })
+              }}
+            />
+            <div style={{ marginRight: 10 }}></div>
+
+            <TextInputField
+              label={t('options.gamescope.upscaleHeight', 'Upscale Height')}
+              htmlId="upscaleHeight"
+              placeholder=""
+              maxLength={4}
+              value={gamescope.upscaleHeight}
+              afterInput={
+                <FontAwesomeIcon
+                  className="helpIcon"
+                  icon={faCircleInfo}
+                  title={t(
+                    'help.gamescope.upscaleHeight',
+                    'The height resolution used by gamescope. A 16:9 aspect ratio is assumed.'
+                  )}
+                />
+              }
+              onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                setGamescope({
+                  ...gamescope,
+                  upscaleHeight:
+                    setResolution(event.currentTarget.value) ??
+                    gamescope.upscaleHeight
+                })
+              }}
+            />
+          </div>
+          {/* Window Type */}
+          <SelectField
+            label={'Window Type'}
+            htmlId="windowType"
+            onChange={(event: ChangeEvent<HTMLSelectElement>) =>
+              setGamescope({
+                ...gamescope,
+                windowType: event.currentTarget.value
+              })
+            }
+            value={gamescope.windowType}
+          >
+            {['fullscreen', 'borderless'].map((opt, i) => (
+              <option key={i}>{opt}</option>
```

Comment:
```suggestion
              <option key={i} value={opt}>
                {t(`gamescope.windowType.${opt}`, opt)}
              </option>
```

We need to add this also somewhere before this return for the `yarn i18next` command:

```
// t('gamescope.windowType.fullscreen', 'Fullscreen')
// t('gamescope.windowType.borderless', 'Borderless')
```

We can do something similar for the upscaling method select, to say `AMD FidelityFX™ Super Resolution` instead of `fsr` for example?